fbw-a380x/src/wasm/systems/a380_systems_wasm/src/lib.rs

```diff
@@ -582,13 +581,9 @@ async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn Error>> {
             );
 
             builder.copy(
-                Variable::aircraft("EXTERNAL POWER AVAILABLE", "Bool", i),
+                Variable::named(&format!("A32NX_EXT_PWR_AVAIL:{i}")),
```

Comment:
Might want to define "A32NX_" as key_prefix i.e.

```rs
let key_prefix = "A32NX_";
...
Variable::named(&format!("{}_EXT_PWR_AVAIL:{}", key_prefix, i)),
```

Reply:
I'm not even sure if the `A32NX_` should be necessary in rs, typically it should not be needed unless I'm missing something.

Reply:
I am honestly unclear how this prefix thing works, so i am just doing what works and cant wait for the day the prefix finally dies

Reply:
For a `named` variable the prefix is automatically added. So correct would be:
```suggestion
                Variable::named(&format!("EXT_PWR_AVAIL:{i}")),
```

Reply:
yep i noticed that now and changed it accordingly